### ISSUE TITLE
Questions about different uncertainty calculation method in pointrend project and original paper

### ISSUE THREAD

**Issue by sunnyheart008205** [facebookresearch/detectron2#901]
I found a question about different uncertainty calculation method in pointrend project and original paper as follows:

In the fuction "calculate_uncertainty" of PointRend project, (detectron2/projects/PointRend/point_rend/roi_heads.py / ) 
 def calculate_uncertainty(logits, classes):
    """
    We **estimate uncerainty as L1 distance between 0.0 and the logit prediction** in 'logits' for the
        foreground class in `classes`.

And in paper 4. Experiments: Instance Segmentation, 
**"We use the distance between 0.5 and the probability of the ground truth class interpolated from the coarse prediction as the point-wise uncertainty measure"**

Q1: I have no idea about the difference here?
I think it makes sense to use the distance between 0.5 and the probability of the ground truth class.

Q2: I used the pretrained pointrend model (PointRend | R50-FPN | 3× | 224×224 )to inference my own images, and found a problem "very poor result appear in oblect hollowed-out body part (like human) ", which exists in all most of  results,
Is this problem related to the uncertainty calculation method or any other trainning/inference problem?

Can you help me? I will preciate it, Thanks

**Comment by ppwwyyxx** [comment]
1. This is a simplified implementation using the fact that sigmoid(0) = 0.5
2. github issues are for discussion about the code and we do not discuss research topics such as how well a model works
